docs/API.md

```diff
@@ -613,6 +613,10 @@ end
 
 - `force` (boolean): if set `true` force terminates the app
 
+### Launch Optionals
```

Comment:
The description for `launch(String appName)` above should also be updated (sorry, GitHub doesn't let me comment there since it wasn't changed).

Reply:
updated.